### DISCUSSION TITLE
Is there any documentation for all the settings?

### DISCUSSION THREAD

**Discussion by gibsonfan2332** [deforum-art/sd-webui-text2video#36] [Q&A] [answered]
I just started playing around with this, but I don't know what all the settings do. Specifically the ones that aren't in the default image generation.

Is there documentation for these somewhere or can they get added as tooltips in the webUI? I don't see anything on the readme about these.

**Comment by jd-3d** [comment]
+1 on this. I'd also like to know why the seed field is grayed out. I'd like to experiment with the same seed to learn more about the settings.

**Comment by kabachuha** [comment, marked as answer]
Give it a bit of time, it's all under construction rn, particularly when the vid2vid mode is to come soon and the settings can change quite much until all of that is settled on

**Reply by jd-3d** [reply]
Thanks @kabachuha. I tried editing the modelscope-text2vid.py to force a seed value of 1 (since its grayed out in the UI) but it still chooses a random seed. Any ideas on that? I'd love to help out by experimenting with various settings on the same seed to see what improves things.

**Reply by kabachuha** [reply]
Absolutely, I can even take on this issue right now

**Reply by gibsonfan2332** [reply]
I assumed as much, just figured I would check in case there was any documentation somewhere. Thanks for your hard work!
